schemas/FeatureManagement.Dotnet.v2.0.0.schema.json

```diff
@@ -1,313 +0,0 @@
-{
```

Comment:
I suppose we'll also need a code update in FeatureManager to not respect variants and telemetry when defined in .NET schema, right?

Reply:
Is it planned as a separate PR?

Reply:
> Is it planned as a separate PR?

Yes.

Reply:
This PR: https://github.com/microsoft/FeatureManagement-Dotnet/pull/421